src/service/limited-partnerships-service.ts

```diff
@@ -0,0 +1,13 @@
+import { logger } from "../utils/logger";
+
+export const createLimitedPartnership = async (
+
+  transactionId: string,
+
+): Promise<string> => {
+  logger.info(`Calling 'postLimitedPartnership' for transaction id '${transactionId}'`);
+
+  // Return a hardcoded string with await
+  return await Promise.resolve("Dummy response");
```

Comment:
You can remove the `await` and `Promise.resolve` wrapper code - they cancel each other out. Just return the string.

Reply:
removed await and Promise wrapper code